docker/Dockerfile

```diff
@@ -18,22 +18,25 @@ RUN apk add --no-cache \
     qt5-qtbase \
     qt5-qtsvg-dev \
     qt5-qttools-dev \
+    qtchooser \
+    qt5-qtbase-dev \
     unzip \
-    wget
-
-# Get, compile and test Iaito from master branch
-RUN git clone --recurse-submodules https://github.com/radareorg/iaito.git /opt/iaito && \
-    git clone --recurse-submodules https://github.com/radareorg/radare2 /opt/iaito/radare2
-
-COPY build_radare2.sh /opt/iaito/
-COPY build_iaito.sh /opt/iaito/
-WORKDIR /opt/iaito
-RUN bash build_radare2.sh
-RUN bash build_iaito.sh && \
-bash -c 'if [[ ! -x "/opt/iaito/build/iaito" ]]; then exit -1; fi'
+    zip \
+    wget && \
+    git clone --recurse-submodules https://github.com/radareorg/iaito.git /opt/iaito && \
+    git clone --recurse-submodules https://github.com/radareorg/radare2 /opt/radare2 && \
+    cd /opt/radare2 && ./sys/install.sh /usr && \
+    cd /opt/iaito && sh configure && make && \
+    bash -c 'if [[ ! -x "/opt/iaito/build/iaito" ]]; then exit -1; fi'
```

Comment:
```suggestion
```

move this check into another RUN, because this `exit` will not exit the RUN step. it will quit the new bash session you are creating here and it wont work